docs/setup/settings.asciidoc

```diff
@@ -517,6 +598,10 @@ back end server. To allow remote users to connect, set the value to the IP addre
 the <<server-socketTimeout, `server.socketTimeout`>> counter. *Default: `"120000"`*
 
 |[[server-maxPayloadBytes]] `server.maxPayloadBytes:`
+ | deprecated:[7.13.0,"This setting will no longer be supported as of 8.0."]
```

Comment:
```suggestion
 | deprecated:[7.13.0,"In 8.0 and later, this setting will no longer be supported."]
```

Reply:
Thanks! There are a couple of other config properties with the same string (`cpu.cgroup.path.override` and `cpuacct.cgroup.path.override`), I'll update those too.